maintainers/testing.md

```diff
@@ -87,6 +87,28 @@ Overall goal, follow the open source ecosystem for infrastructure choices
     - I.E. cuDF PyTests:
       - [https://github.com/rapidsai/cudf/tree/branch-0.7/python/cudf/tests](https://github.com/rapidsai/cudf/tree/branch-0.7/python/cudf/tests)
 
+## Datasets
+
+- Many tests depend on the presence of specific datasets in order to properly verify code correctness.
```

Comment:
I don't think this whole section should be a bulleted list. Maybe take the first two bullets, condense them down and put under `## Datasets` as an overview sentence or two.

Reply:
Agreed. I incorporated your suggestion in the separate document that Mike mentioned I should create.